client/scss/components/_status-tag.scss

```diff
@@ -4,6 +4,7 @@
   border-radius: 2px;
   text-align: center;
   display: inline-block;
+  flex-shrink: 0;
```

Comment:
As discussed we’ll change this to `word-break: normal` to fix #11875 while reducing the risk of page titles being squashed by long locale names (`Portuguese (Brazil) (latest)`).